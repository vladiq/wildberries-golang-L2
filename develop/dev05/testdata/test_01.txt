aadaadaklsd
addsADasd
34r43r
fsdsfndms aDDsadasd
sfdshjfjskfdsj addSADasd
fsdnfsflsdf
sdfsfsdaddsadasd