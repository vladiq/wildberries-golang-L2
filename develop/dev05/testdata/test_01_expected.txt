1 addsADasd
3 fsdsfndms aDDsadasd
4 sfdshjfjskfdsj addSADasd
6 sdfsfsdaddsadasd

